packages/odyssey-react-mui/src/Select.tsx

```diff
@@ -272,7 +272,8 @@ const Select = <
               />
             )}
             {option.text}
-            {internalSelectedValues === option.value && (
+            {(internalSelectedValues?.includes(option.value) ||
+              internalSelectedValues === option.value) && (
```

Comment:
I'm fine with this code, but I think it'd be better to make it clear which case would have an array and which wouldn't by using a ternary with `isMultiSelect`:

```suggestion
            {(isMultiSelect ? internalSelectedValues.includes(option.value) :
              internalSelectedValues === option.value) && (
```

Reply:
@KevinGhadyani-Okta Even in the non-multiselect version the value is sometimes an array. We need to check both cases I think